### DISCUSSION TITLE
New variant for lgCard

### DISCUSSION THREAD

**Discussion by NemirStephan** [Legal-and-General/canopy#277] [Ideas] [answered]
@alexcanning has suggested we should reduce the amount of drop shadows on our pages. Because we are now using LgCard extensively, drop shadows are becoming the norm rather than an exception to draw customers attention to important information.

I'd like to propose we add a new variant input to LgCard as follows
| Name.     | Description                                                       | Type                 | Default     | Required |
| --------- | ------------------------------------------- | -------------- | ---------- | --------- |
| `variant` | The variant of the card : `flat` or `shadow` | LgCardVariant | `shadow` | No            |

I'm suggesting the default should be `shadow` for backwards compatibility

@andy-polhill  and @owensgit, what are your thoughts on this?

**Comment by owensgit** [comment, marked as answer]
We discussed this on a call recently, as it's also needed elsewhere.

The consensus was that we could create a util directive for the drop shadow, similar to the `lgMargin` and `lgPadding` directives, which can apply either style.

For example
`<lg-card lgShadow="none">`

Potentially there could be a choice of shadow "depths", e.g. `lgShadow="sm"`, `lgShadow="md"`, or something to that effect.

**Reply by owensgit** [reply]
The shadow directive was created and can be found here:

_**src/canopy/projects/canopy/src/lib/shadow/shadow.directive.ts**_
